<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenHysteria</class>
 <widget class="QWidget" name="GenHysteria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Certificate Path</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Obfs Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="down_mbps">
     <property name="text">
      <string notr="true">50</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string notr="true">down_mpbs</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="gen">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="MyLineEdit" name="auth_string"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string notr="true">up_mbps</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="protocol">
     <item>
      <property name="text">
       <string notr="true">udp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">wechat-video</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="MyLineEdit" name="obfs_password"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="up_mbps">
     <property name="text">
      <string notr="true">10</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string notr="true">SNI</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="MyLineEdit" name="cert_path"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="alpn"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string notr="true">ALPN</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="MyLineEdit" name="sni"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Auth String</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="allow_insecure">
     <property name="text">
      <string>Allow Insecure</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/HyNetwork/hysteria/wiki&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;Document&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/widget/MyLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>obfs_password</tabstop>
  <tabstop>auth_string</tabstop>
  <tabstop>protocol</tabstop>
  <tabstop>up_mbps</tabstop>
  <tabstop>down_mbps</tabstop>
  <tabstop>sni</tabstop>
  <tabstop>alpn</tabstop>
  <tabstop>cert_path</tabstop>
  <tabstop>gen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
